src/lib/components/Map/map.tsx

```diff
@@ -109,17 +118,20 @@ export const Map: React.FC<MapPropsType> = (props) => {
 
     return (
         <div ref={mapRef} className="Map" style={{ width: width, height: height }}>
-            <View
-                initialCenterPoint={viewCenterPoint}
-                width={width}
-                height={height}
-                Scene={Scene}
-                boundaryBox={boundaryBox}
-                onCenterPointChange={handleViewCenterPointChange}
-                scale={scale}
-                id={id}
-                backgroundColor={config.backgroundColor}
-            />
+            {id !== previousId ? (
+                <></>
+            ) : (
```

Comment:
If id changes we should not render View? Why?

Reply:
It was implemented in order to keep the animation from behaving weirdly. However, I just wanted to check what exactly happened (has been a while) when I remove it. I did not see any effect, so seems like this is no longer required. I'm gonna remove it.